backend/bin/kafkaConsumer/exerciseConsumer/handleMessage.ts

```diff
@@ -2,15 +2,19 @@ import { Message } from "./interfaces"
 import { saveToDatabase } from "./saveToDB"
 import { Prisma } from "../../../generated/prisma-client"
 import { MessageYupSchema } from "./validate"
+import { Mutex } from "await-semaphore"
+import { Logger } from "winston"
+import { KafkaConsumer } from "node-rdkafka"
+
 const config = require("../kafkaConfig.json")
 
 let commitCounter = 0
 const commitInterval = config.commit_interval
 export const handleMessage = async (
-  kafkaMessage,
-  mutex,
-  logger,
-  consumer,
+  kafkaMessage: any, // TODO: find type
```

Comment:
The interface definition can be found here: https://github.com/rage/mooc.fi/blob/master/backend/docs/kafka.md#user-course-progress